src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,128 @@
+import { useContext } from 'react';
+import { PersonItem } from '../PersonItem';
+import { PeopleContext } from '../../peopleContext';
+import { useNavigate, useSearchParams } from 'react-router-dom';
+import { SortBy, SortOrder, SortQueries } from '../../types/Sort';
+import { Person } from '../../types';
+import { sortPeople } from '../../utils/sortPeople';
+
+export const PeopleTable: React.FC = () => {
+  const [searchParams] = useSearchParams();
+  const filteredPeople = useContext(PeopleContext);
+  const navigate = useNavigate();
+
+  const sortByField = searchParams.get(SortQueries.sort);
+  const sortOrder = searchParams.get(SortQueries.order) || SortOrder.off;
+
+  if (!filteredPeople.length) {
+    return;
```

Comment:
Here you are returning nothing if there are no filtered people. This might lead to unexpected behavior. It's better to return some placeholder or message indicating that there are no people to display.